.github/workflows/pr.yml

```diff
@@ -58,6 +58,7 @@ jobs:
             { java: 8, gradle: 6.8 },
             { java: 17, gradle: 7.3.1 }
         ]
+      max-parallel: 1
```

Comment:
I bumped this down to 1 but I'm still seeing issues in CI.

Reply:
Great, I like this change!

Reply:
Also you said you were seeing issues in CI so what change resolved those issues? I see that the existing tests have been kept as it is.